Formula/busco.rb

```diff
@@ -15,17 +15,57 @@ class Busco < Formula
     sha256 "f77caa27c9ea8e866682a00cd93bbb626f6139b6a95e1c7354b0268ad806a253" => :x86_64_linux
   end
 
+  depends_on "augustus"
+  depends_on "hmmer"
   depends_on "python"
   depends_on "blast" => :recommended
-  # Also depends on augustus and hmmer
+  depends_on "r" => :optional
 
   def install
     inreplace Dir["scripts/*.py"], "#!/usr/bin/env python", "#!#{HOMEBREW_PREFIX}/bin/python3"
     system "python3", "setup.py", "install", "--prefix=#{prefix}"
-    bin.install Dir["scripts/*"]
-    bin.install_symlink "run_BUSCO.py" => "busco"
+
+    libexec.install Dir["scripts/*"]
+    (bin/"busco").write_env_script(libexec/"run_BUSCO.py", :AUGUSTUS_CONFIG_PATH => "#{Formula["augustus"].prefix}/config/")
+
     doc.install "BUSCO_v3_userguide.pdf"
-    prefix.install "config/config.ini.default"
+    prefix.install "config"
+
+    config_ini = <<~EOS
+      [busco]
+
+      [tblastn]
+      path = #{Formula["blast"].bin}
+
+      [makeblastdb]
+      path = #{Formula["blast"].bin}
+
+      [augustus]
+      path = #{Formula["augustus"].bin}
+
+      [etraining]
+      path = #{Formula["augustus"].bin}
+
+      [gff2gbSmallDNA.pl]
+      path = #{Formula["augustus"].prefix}/scripts/
+      [new_species.pl]
+      path = #{Formula["augustus"].prefix}/scripts/
+      [optimize_augustus.pl]
+      path = #{Formula["augustus"].prefix}/scripts/
+
+      [hmmsearch]
+      path = #{Formula["hmmer"].bin}
+    EOS
+
+    if build.with? "r"
+      config_ini += <<~EOS
+
+        [Rscript]
+        path = #{Formula["r"].bin}
```

Comment:
```suggestion
        path = #{HOMEBREW_PREFIX}
```

Reply:
Will `brew cask install r` go into `HOMEBREW_PREFIX` ?

Reply:
Yes.
```
❯❯❯ realpath /usr/local/bin/R
/Library/Frameworks/R.framework/Versions/3.4/Resources/bin/R
```

Reply:
Cool. I use brew r (and am quite happy with it :P). But I'm happy to follow conventions.

Reply:
This should be
```
path = #{HOMEBREW_PREFIX}/bin
```
correct?

Reply:
Yep!

Reply:
> Cool. I use brew r (and am quite happy with it :P). But I'm happy to follow conventions.
 
The cask `r` is able to use the precompiled binary R packages from CRAN, but the brewed `r` cannot.